src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -160,7 +161,7 @@ def skimToDataTier(cmsswVersion, skim):
         command += "cd %s\n" % scramBaseDirs[0]
         command += "eval `scramv1 runtime -sh`\n"
 
-        if PY3:
+        if isCMSSWSupported(cmsswVersion, "CMSSW_10_3_0"):
```

Comment:
Is this the exact version from which on this `getSkimDatatier` method is fixed to support python3?

Reply:
Hi Todor. What I did is I looked for the PR that made the changes: https://github.com/cms-sw/cmssw/pull/23932
It was introduced in CMSSW_10_3_0_pre1, but I think `isCMSSWSupported()` does not handle pre-releases.

Reply:
Hi @germanfgv I think this check is pretty much enough for the current situation. Thanks!

Reply:
That's correct, this method doesn't go beyond the 3rd digit of the release number. 
Thanks for this confirmation, German, I missed your in-thread reply :)